app/lib/yahoo_finance.rb

```diff
@@ -0,0 +1,98 @@
+require 'yahoofinance'
+require 'json'
+
+# Gets stock data from Yahoo Finance
+class Finance
+  # Looks up symbol in the finance cache. Historical data is current for up to 
+  # a day in the cache and current data for 15 minutes. After that new data 
+  # must be fetched
+  def self.get_quotes(symbol, hist)
+    symbol.upcase!
+    if Company.find_by(symbol: symbol).nil?
+      fail(ArgumentError,
+           "#{symbol} is an invalid symbol! Not found in company database")
+    end
+
+    #cached_result = get_cached_data(symbol, hist)
+    if hist 
+      return fetch_finance_hist(symbol)# unless cached_result
```

Comment:
Looking through my code,it looks like rubocop didn't complain for me either. I actually don't think that's extraneous now that I think about it when it's not late at night. Sorry :)  But you know how ruby has implicit returns?  So you usually don't need to write return but you do here after all otherwise it'd be ambiguous